server/routes/apiRoutes.js

```diff
@@ -12,83 +12,154 @@ var db = require("../models");
   });
 
-  app.post("/api/user", function (req, res) {
+  //create a user account
+  app.post("/api/user", function (req, res) 
+  {
     console.log(req.body)
-    db.User.create({
-      email: req.body.userEmail,
-      password: req.body.userPassword
+    db.user.create(
+      {
+      fname: req.body.fname,
+      lname: req.body.lname,
+      email: req.body.email,
+      password: req.body.password
+      })
+      .then(function (dbuser) 
+      {
+        res.json(dbuser);
+      });
+  });
 
-    })
-      .then(function (dbUser) {
-        res.json(dbUser);
+  //rcreate a post
+  app.post("/api/event_post", function(req, res)
+  {
+    db.post.create(
+      {
+        poster_user_id: req.body.poster_user_id,
+        title: req.body.title,
+        content: req.body.content,
+        status: 'In Review'
+      })
+      .then(function(dbpost)
+      {
+        res.json(dbpost)
       });
   });
 
-  app.post("/api/events", function (req, res) {
-    console.log(req.body)
-    db.Event.create({
+  //create a comment
+  app.post("/api/post_comment", function(req,res)
+  {
+    db.comment.create(
+    {
+      post_id: req.body.post_id,
+      commenter_user_id: req.body.commenter_user_id,
       title: req.body.title,
-      description: req.body.description,
-      date: req.body.date,
-      time: req.body.time,
-      address: req.body.address,
-      name: req.body.name
-
+      content: req.body.content,
+      status: 'In Review'
     })
-      .then(function (dbEvent) {
-        res.json(dbEvent);
-      });
+    .then(function(dbcomment)
+    {
+      res.json(dbcomment)
+    });
   });
 
-  app.get("/api/events", function (req, res) {
-    db.Event.findAll({}).then(function (dbEvent) {
-      res.json(dbEvent);
+  //create event
+  app.post("/api/events", function (req, res) 
+  {
+    console.log(req.body)
+    db.event.create(
+      {
+      //creator_user_id: <needs pass in from auth>,
+      event_title: req.body.title,
+      summary: req.body.summary,
+      max_attend: req.body.max_attend,
+      is_full: 'No',
+      age_restrict: req.body.age_restrict,
+      age_min: req.body.age_min,
+      event_date: req.body.event_date,
+      event_time: req.body.event_time,
+      location_name: req.body.location_name,
+      addr1: req.body.addr1,
+      addr2: req.body.addr2,
+      addr3: req.body.addr3,
+      city: req.body.city,
+      state: req.body.state,
+      zip_code: req.body.zip_code,
+      country: req.body.country
+      })
+      .then(function (dbevent) 
+      {
+        res.json(dbevent);
+      })
+  });
+
+  //list of all events
+  app.get("/api/events", function (req, res) 
+  {
+    db.Event.findAll({})
+    .then(function (dbevent) 
+    {
+      res.json(dbevent);
     });
   });
 
-  app.get("/api/events/:title", function (req, res) {
-    db.Event.findAll({
-      where: { title: req.params.title }
-    }).then(function (dbEvent) {
-      res.json(dbEvent);
+  //get event details by title
```

Comment:
Personally don't like comments. They have to be updated separately as the code changes. It's pretty obvious what this route is doing simply by its URL. Consider removing all of these.